GenderPayGap.WebUI/Services/DraftReturnService.cs

```diff
@@ -61,6 +61,8 @@ private DraftReturn CreateDraftReturnBasedOnSubmittedReturn(Return submittedRetu
             {
                 OrganisationId = submittedReturn.OrganisationId,
                 SnapshotYear = submittedReturn.AccountingDate.Year,
+                
+                OptedOutOfReportingPayQuarters = submittedReturn.OptedOutOfReportingPayQuarters,

```

Comment:
why didn't this issue happen when editing the figures first?

Reply:
We explicitly save the opt out of reporting boolean when we edit the figures